src/PrestaShopBundle/Form/Validator/Constraints/TinyMceMaxLengthValidator.php

```diff
@@ -27,25 +27,51 @@
 namespace PrestaShopBundle\Form\Validator\Constraints;
 
 use PrestaShop\PrestaShop\Adapter\LegacyContext;
+use PrestaShop\PrestaShop\Adapter\Validate;
 use Symfony\Component\Validator\Constraint;
 use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
 
 class TinyMceMaxLengthValidator extends ConstraintValidator
 {
+    private $validateAdapter;
+
+    public function __construct()
+    {
+        $this->validateAdapter = new Validate();
+    }
```

Comment:
```suggestion
    public function __construct(Validate $validate)
    {
        $this->validateAdapter = $validate;
    }
```

You have to use dependency injection :)